packages/astro/performance/content-benchmark.mjs

```diff
@@ -25,15 +23,15 @@ async function benchmark({ fixtures, templates, numPosts }) {
 			ext: extByFixture[fixture],
 			template: templates[fixture],
 		});
-		console.log(`[${fixture}] Generated posts`);
+		console.info(`[${fixture}] Generated posts`);
```

Comment:
Is the rename from `console.log` to `console.info` intentional? Seems to have changed for many other code too.

Reply:
Many of these changes had suppression comments (many), this means those logs were intentional. `console.log` are _usually_ for debugging purposes, but `console.info` are usually on purpose.

I deduced that all those logs were actually intentional, which means it makes more sense to turn them into `.info`.

What do you think? Does it make sense? Do you agree?

Reply:
That makes sense to me. I also noticed the PR doesn't really change to `.info` a lot, only for this `performance/` directory which we don't use much either. So I think this is fine by me.